src/NuGet.Clients/NuGet.VisualStudio.Common/Etw/NuGetExtensibilityEtw.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics.Tracing;
+
+namespace NuGet.VisualStudio.Etw
+{
+    public static class NuGetExtensibilityEtw
+    {
+        public static EventSource EventSource { get; } = new EventSource("NuGet-VS-Extensibility");
```

Comment:
I am under the impression that all VS ETW events names have prefix `VS--***`. Should we follow the same convention?

```suggestion
        public static EventSource EventSource { get; } = new EventSource("VS-NuGet-Extensibility");
```

Reply:
Putting a breakpoint in `ExtensibilityTelemetryCollector.ExtensibilityEventListener.OnEventSourceCreated`, I see `eventSource.Name` with values like:

* VisualStudioXamlTools
* Microsoft-VisualStudio-Threading
* Microsoft-VisualStudio-Common
* StreamJsonRpc
* Microsoft-VisualStudio
* Microsoft-VSS-Http
* VssPerformanceEventSource
* Microsoft-VisualStudio-CpsCore
* Microsoft-VisualStudio-CpsVs
* TypeScriptEventSource

This is excluding all the obviously non-VS event sources (StreamRpcJson and MIcrosoft-VisualStudio-Threading are designed to be reusable by other apps, so arguably should be removed from my list. Anyway, none of them start with `VS-`.

Since NuGet is owned by the .NET Foundation, not Microsoft, I thought it seemed more appropriate to make the event source name focus more on NuGet, rather than VS.

Reply:
Ok, sounds good.